### PR TITLE
REL: Release notes v8.5.0

### PR DESCRIPTION
**Pull request by tangkong**
## Description
- Compiles release notes
- modifies conda recipe to use bluesky-base, instead of matplotlib

## Motivation and Context
Note: conda-forge has a pint dependency that we don't have here.  We took this out https://github.com/pcdshub/pcdsdevices/pull/1253 

## How Has This Been Tested?
CI

## Where Has This Been Documented?
This PR, and this is docs

## Pre-merge checklist
- [x] Code works interactively
- [x] Code contains descriptive docstrings, including context and API
- [x] New/changed functions and methods are covered in the test suite where possible
- [x] Test suite passes locally
- [x] Test suite passes on GitHub Actions
- [x] Ran docs/pre-release-notes.sh and created a pre-release documentation page
- [x] Pre-release docs include context, functional descriptions, and contributors as appropriate

### CODE REVIEW

**Comment by tangkong** [comment]
I have requested reviews from all contributors to this tag, for the sake of spot checking our release notes.  Don't feel obligated to give a detailed review, but if you would like to refine any of the wording please leave a review!

**Review by vespos — APPROVED** [review]

**Review by nrwslac — APPROVED** [review]
lgtm!

**Review by slactjohnson — APPROVED** [review]

**Review by baljamal — DISMISSED** [review]

File: pcdsdevices/_version.py
Comment (ZLLentz) [inline pcdsdevices/_version.py:1]: I think this was accidentally comitted

File: docs/source/releases.rst [hunk 1]
```diff
@@ -1,6 +1,112 @@
...
+- Allow motion uis to expand vertically once this functionality gets added to typhos.
+- Use sympy instead of pint for unit conversions for simpler
+  maintainability.
+- Various CI tweaks due to numpy 2.0's chaos.
+- Modifies entrypoint tests to be forward-compatible with py3.12 entrypoint API.
+- Unpins numpy in CI, build incompatibility has been fixed upstream.
+
+Contributors
+------------
+- tongju12
```
Comment (ZLLentz) [inline docs/source/releases.rst:96]: Duplicate contributor name, tong is included below too (in alphabetical order)
```suggestion
```

File: docs/source/releases.rst [hunk 2]
```diff
@@ -1,6 +1,112 @@
 Release History
 ###############
 
+
+v8.5.0 (2024-08-22)
+===================
+
+Features
```
Comment (ZLLentz) [inline docs/source/releases.rst:8]: I'm considering whether this should be changed from "Features" to "Library features" (new components, base classes, utils, etc.) to make it more clear that the device-specific features go in "Device Updates"

**Review by ZLLentz — CHANGES_REQUESTED** [review]
I'll put the request changes marker on because the _version.py file does break things
I'm happy to push a commit to rearrange releases.rst

File: pcdsdevices/_version.py
Comment (tangkong) [reply pcdsdevices/_version.py:1]: oh no

**Review by ZLLentz — APPROVED** [review]

**Pull request merged**